services/board/src/Domain/MovingStrategies/Moving.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace CliChess\Board\Domain\MovingStrategies;
+
+use CliChess\Board\Domain\Square;
+
+interface Moving
+{
+    public function canMove(Square $from, Square $to): bool;
```

Comment:
Not very satisfied with the naming of this method actually!
Since it's a strategy, how about `applicable` or `movable`? Or something else? Any idea?

Reply:
I also think this is not ideal, but let's leave it this way for now because I guess we will find a better name once we need to return the trajectory.